lib/http-routes/api/create-call.js

```diff
@@ -293,6 +293,8 @@ router.post('/',
           },
           cbProvisional: (prov) => {
             const callStatus = prov.body ? CallStatus.EarlyMedia : CallStatus.Ringing;
+            // Update call-id for sbc outbound INVITE
+            cs.callInfo.sipCallid = prov.get('X-CID');
```

Comment:
can we rename this `sbcCallId` everywhere

Reply:
Fixed to use sbcCallid